contracts/periphery/contracts/s2s/examples/SystemRemarkDemoApp.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.6.0;
+
+import "../SmartChainApp.sol";
+import "@darwinia/contracts-utils/contracts/Scale.types.sol";
+
+// contract SystemRemarkDemoApp is PangolinToPangoroApp {
+contract SystemRemarkDemoApp is SmartChainApp {
+
+	function systemRemark() public payable {
+		// 1. prepare the message
+        //    this can be extract to a method:
+        //        Types.Message message = Pangoro.buildSystemRemarkCallMessage(hex"12345678");
+		System.RemarkCall memory call = System.RemarkCall(
+            hex"0001",
```

Comment:
Is this pallet index encoded?

Reply:
the call index does not need to scale encode. 
another demo generates the same result.